org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/template/java/JavaPostfixContext.java

```diff
@@ -322,6 +323,10 @@ public boolean canEvaluate(Template template) {
 			return false;
 		}
 
+		if (selectedNode instanceof Javadoc) {
```

Comment:
I would combine w/ check on L318

Reply:
Makes sense, since the check technically depends on that. In the future we could always move this into a separate method if we end up needing a bunch of different node checks to enable/disable postfix completion.

Reply:
Also, it's not just `Javadoc` you need to catch, it's `TagElement` as well.

Try with :
```
/**
 * {@link ArrayList}
 * test.
 */
```

Reply:
Thank you for catching that. I changed the return value of `public boolean visit(Javadoc node)` to `false`, to stop visiting the child, then we can handle this case.